app/screens/post_options/index.ts

```diff
@@ -146,8 +147,9 @@ const enhanced = withObservables([], ({combinedPost, post, showAddReaction, sour
         }),
     );
 
-    const canDelete = combineLatest([canDeletePostPermission, channelIsArchived, channelIsReadOnly, canPostPermission]).pipe(switchMap(([permission, isArchived, isReadOnly, canPost]) => {
-        return of$(permission && !isArchived && !isReadOnly && canPost);
+    const canDelete = combineLatest([canDeletePostPermission, channelIsArchived, channelIsReadOnly, canPostPermission, currentUserId]).pipe(switchMap(([permission, isArchived, isReadOnly, canPost, userId]) => {
+        const canDeleteBoRPost = borPost ? post.userId === userId : false;
```

Comment:
shouldn't this be true? I would even not use ternary expression for clarity since this BoR logic changes everything.

Reply:
not related to this change, but wondering if are there actually tests for regular post deletion?